about/shared-responsibility.md

```diff
@@ -0,0 +1,63 @@
+# Shared Responsibility Model
+
+Many things must be done to successfully run a hub for a community.
+Some of them are content-focused, some are community-focused, others are infrastructure-focused.
+
+2i2c's aim is to **share responsiblity for each hub** with the communities we serve. We do this by defining the responsibilities that are a good fit for the skills and goals of each organization.
+This "Shared Responsibility Model" is a useful way to understand what actions communities are still expected to perform under a service agreement with 2i2c.[^1]
+
+[^1]: This is inspired by the [Shared Responsibility Model](https://aws.amazon.com/compliance/shared-responsibility-model/) from Amazon Web Services.
+
+## Why follow this model
+
+We use a Shared Responsibility model because we believe that it leads to the best service for the communities that we serve.
+Our main goal is to ensure that each hub service is maximally impactful for the community it serves, and that it achieves this impact in the most efficient way possible.
+
+To do that, we want to do two things:
+
+- Assign responsibilities that are well-suited to the skills and the interests of each group. This will boost the impact of each party.
+- Choose the operational model that is most well-matched to whomever is carrying out a responsibility. This will boost the efficiency of each party.
+
+Our challenge is to figure out which responsibiltiies should lie with the community, and which should lie with the 2i2c team, to strike this balance of impact and efficiency.
+
+The other reason we follow this model is that it helps us follow the [Community Right to Replicate](https://2i2c.org/right-to-replicate) their infrastructure. We think of our community hubs as being a collaboration between 2i2c and the communities that use it, and this framing helps us be explicit about where we fit into the picture.
```

Comment:
This first sentence doesn't quite make sense to me. Is it supposed to say "it helps us _allow_ the Community Right to Replicate their infrastructure"?